server/sonar-db-dao/src/test/java/org/sonar/db/qualityprofile/ActiveRuleDaoTest.java

```diff
@@ -70,6 +70,7 @@
   private RuleDefinitionDto rule1 = RuleTesting.newDto(RuleTesting.XOO_X1).getDefinition();
   private RuleDefinitionDto rule2 = RuleTesting.newDto(RuleTesting.XOO_X2).getDefinition();
   private RuleDefinitionDto rule3 = RuleTesting.newDto(RuleTesting.XOO_X3).getDefinition();
+  private RuleDefinitionDto rule4 = RuleTesting.newRule().setStatus(RuleStatus.REMOVED);
```

Comment:
I'm not a big fan of objects shared between tests (including related @Before methods). However `rule4` should be renamed `removedRule` to prevent ambiguity. It allows to drop the comments `/* status=removed */` everywhere the field is used.